bin/validate_ssa.py

```diff
@@ -92,17 +104,16 @@ def test_detection(test, args):
     with open(test, 'r') as fh:
         test_desc = yaml.safe_load(fh)
         name = test_desc['name']
-        print("Testing %s" % name)
+        log(logging.INFO, "Testing %s" % name)
```

Comment:
It's more usual to specify `name` as an additional argument. The logger will fill it for you:
`log(logging.INFO, 'Testing %s', name)`

(If you attached a log handler which sent the logs to Splunk or something then it would be possible to break out your `name` parameter separately.)